.github/workflows/pull_request.yaml

```diff
@@ -54,6 +54,48 @@ jobs:
           verbose: true
           fail_ci_if_error: false
 
+  functional-tests:
+      runs-on: ubuntu-latest
+      timeout-minutes: 15
+      steps:
+        - uses: actions/checkout@v3
+
+        - uses: actions/setup-go@v3
+          with:
+            go-version-file: './go.mod'
+
+        - id: go-cache-paths
```

Comment:
This isn't needed with the other change, remove in a subsequent PR to get this one in @jon-whit